server/schemas/typeDefs.js

```diff
@@ -65,7 +65,7 @@ const typeDefs = gql`
     type Mutation {
         addUser(
             business_id: ID!
-            business_brand_name: String!
+            brand_name: String!
```

Comment:
Updating to match already existing name so easier to work with everything and removing complexity